### ISSUE TITLE
Typechecking bug

### ISSUE THREAD

**Issue by huewang** [freshollie/jest-dynalite#29]
On jest-dynalite v3.2.4, when I attempted to typecheck files using jest-dynalite I got the error:
```
node_modules/jest-dynalite/dist/db.d.ts:1:22 - error TS6053: File '[redacted]/node_modules/jest-dynalite/types.d.ts' not found.

1 /// <reference path="../types.d.ts" />
                       ~~~~~~~~~~~~~

node_modules/jest-dynalite/dist/db.d.ts:2:59 - error TS2694: Namespace '"*"' has no exported member 'DynaliteServer'.

2 export declare const dynaliteInstance: import("dynalite").DynaliteServer;
```

Version 3.2.3 works fine though.
Did something in https://github.com/freshollie/jest-dynalite/commit/e6a3ab4ca720710f7829e1b8fc770ffb28cbbe40 break version 3.2.4?

jest-dynalite has been very useful for our dev team by the way and we appreciate the effort put into its development!

**Comment by freshollie** [comment]
I would add `skipLibCheck` to your `tsconfig`. There isn't really a need to run type checking against external libraries.

**Comment by freshollie** [comment]
I'll put out a fix for this anyway

**Comment by freshollie** [comment]
Fix should be released shortly as `v3.2.5` :)
